src/components/DisplayNames/index.native.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Text from '@components/Text';
+import DisplayNamesProps from './types';
+
+// As we don't have to show tooltips of the Native platform so we simply render the full display names list.
+function DisplayNames({accessibilityLabel, fullTitle, textStyles = [], numberOfLines = 1}: DisplayNamesProps) {
+    return (
+        <Text
+            accessibilityLabel={accessibilityLabel}
+            style={textStyles}
+            numberOfLines={numberOfLines}
+        >
+            {fullTitle}
```

Comment:
fullTitle is an optional prop. Should we add a condition to render this component or maybe this prop should be required? I'm not sure if we should render an empty Text component.

Reply:
ok :)